pkg/install/install.go

```diff
@@ -124,8 +124,8 @@ func NewInstaller(ctx context.Context, log *logrus.Entry, env env.Interface, db
 		env:          env,
 		db:           db,
 		billing:      billing,
-		cipher:       cipher,
 		doc:          doc,
+		cipher:       cipher,
```

Comment:
Here I just made the order consistent with that in the `Installer` struct type definition